cscs-checks/cuda/cuda_memtest_check.py

```diff
@@ -0,0 +1,46 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class FlexibleCudaMemtest(rfm.RegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.valid_systems = ['daint:gpu', 'dom:gpu']
+        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
+                                    'PrgEnv-intel', 'PrgEnv-pgi']
+        self.descr = 'Flexible Cuda Memtest'
+        self.executable = 'osu_alltoall'
+        self.maintainers = ['TM', 'VK']
+        self.tags = {'maintenance'}
```

Comment:
Don't put a tag here. Otherwise it will run on every maintenance session on all the nodes!